src/modules/commentPopUp.js

```diff
@@ -0,0 +1,130 @@
+/* eslint-disable guard-for-in */
+import axios from 'axios';
+
+import getComments from './commentApi';
+
+const popDiv = document.getElementById('popDiv');
+const showCommentModal = (movie) => {
+  popDiv.style.display = 'flex';
+  const popUpContainer = document.createElement('div');
+  popUpContainer.classList.add('popUpContainer');
+
+  const popUpTopDiv = document.createElement('div');
+  popUpTopDiv.classList.add('popUpTopDiv');
+
+  const movieImg = document.createElement('img');
+  movieImg.classList.add('movieImg');
+  movieImg.setAttribute('src', movie.image.original);
+
+  const popUpCloseBtn = document.createElement('i');
+  // popUpCloseBtn.innerText = 'colse';
+  popUpCloseBtn.classList.add('popUpCloseBtn', 'fa-solid', 'fa-close', 'closeIcon');
+  popUpCloseBtn.addEventListener('click', () => {
+    popDiv.style.display = 'none';
+    // document.location.reload;
```

Comment:
- [ ] OPTIONAL
There are commented code blocks found in your code on the repository. Kindly endeavor to remove **all** commented code from your code before pushing it to the repository so as to keep your code well organized.

Reply:
Thanks for pointing that out. The comments have been removed.

https://github.com/iamchristianani/js-capstone/blob/febf719dac18cfc2152e1eb6f39eb294d7f304a2/src/modules/commentPopUp.js#L20-L24